### DISCUSSION TITLE
Doris四舍五入问题

### DISCUSSION THREAD

**Discussion by jwsmai** [apache/doris#14645] [A - General / Q&A]
<img width="397" alt="image" src="https://user-images.githubusercontent.com/63184217/204248109-9c907f2a-fcba-4ac7-9d15-b67452dc2071.png">
为啥0.25保留1位小数是0.2，不应该是0.3吗？

**Comment by LemonLiTree** [comment]
这是哪个版本？目前1.1,master版本都是0.3
<img width="449" alt="image" src="https://user-images.githubusercontent.com/109049295/204259774-dc10026f-5836-4dad-ab8f-91c49da7c3bb.png">

**Reply by jwsmai** [reply]
@LemonLiTree 厂商托管的
<img width="262" alt="image" src="https://user-images.githubusercontent.com/63184217/204261318-271c7876-d387-424a-9e74-b7d40d931435.png">

**Comment by LemonLiTree** [comment]
show variables where variable_name = 'version_comment';
看下

**Reply by jwsmai** [reply]
<img width="588" alt="image" src="https://user-images.githubusercontent.com/63184217/204538494-9be94e8b-c76e-44a3-b216-9911c2c840eb.png">

**Reply by LemonLiTree** [reply]
应该是旧版本的bug，建议升级
